javascripts/scroll-up.js

```diff
@@ -0,0 +1,20 @@
+export default function () {
+  // function to scroll up to page top
+  const PageTopBtn = document.getElementById('js-scroll-top')
+  PageTopBtn.addEventListener('click', (e) => {
```

Comment:
Hey @kazuhitonakayama ,

I'm working to get this change deployed, but one of the tests are failing. I don't have access to your branch could you add something like

`if(!PageTopBtn) return`

before the `addEventListener` call?